packages/core/data-transfer/lib/engine/__tests__/engine.test.ts

```diff
@@ -0,0 +1,390 @@
+import { ObjectWritableMock, ObjectReadableMock } from 'stream-mock';
```

Comment:
What does this library offer in terms of features we don't have if we declare custom readable/writable streams?

Reply:
Considering what I'm using it for, it saved me writing a "write" stub... in exchange for having to manually destroy the streams. I've killed it and just used readbable/writable instead :)